logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/user/UpdateLoggedInUsersCountUseCase.kt

```diff
@@ -0,0 +1,35 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ */
+package com.wire.kalium.logic.feature.user
+
+import com.wire.kalium.logic.data.AccessRepository
+
+/**
+ * This use case will update the number of logged in users
+ */
+interface UpdateLoggedInUsersCountUseCase {
+    suspend operator fun invoke(count: Int)
+}
+
+class UpdateLoggedInUsersCountUseCaseImpl internal constructor(
```

Comment:
keeping track of logged in users this way is very fragile and error-prone, and tbhI don't see the befits worth the risk